.github/actions/prefix-pr/action.yml

```
@@ -1,9 +1,5 @@
-name: "Prefix PR with issue key"
-description: "Prefixes the PR title with a Jira issue key, and also adding the key to the body if not present"
-
-inputs:
-  GITHUB_TOKEN:
-    required: true
+name: Validate Jira Issue Code in Pull Request
+description: Prefixes the PR title with a Jira issue key, and also adds the key to the body if not present
```

As I [pointed this before](https://github.com/Lombiq/GitHub-Actions/pull/90#discussion_r1028417820):

```suggestion
description: Prefixes the PR title with a Jira issue key, and also adds the key to the body if not present.
```